src/solvers/base_solver.cpp

```diff
@@ -13,39 +13,41 @@
 #include "common/serac_types.hpp"
 #include "fmt/fmt.hpp"
 
-BaseSolver::BaseSolver(MPI_Comm comm) : m_comm(comm), m_output_type(serac::OutputType::VisIt), m_time(0.0), m_cycle(0)
+namespace serac {
+
+BaseSolver::BaseSolver(MPI_Comm comm) : comm_(comm), output_type_(serac::OutputType::VisIt), time_(0.0), cycle_(0)
 {
-  MPI_Comm_rank(m_comm, &m_mpi_rank);
-  MPI_Comm_size(m_comm, &m_mpi_size);
-  SetTimestepper(serac::TimestepMethod::ForwardEuler);
-  m_order = 1;
+  MPI_Comm_rank(comm_, &mpi_rank_);
+  MPI_Comm_size(comm_, &mpi_size_);
+  setTimestepper(serac::TimestepMethod::ForwardEuler);
+  order_ = 1;
 }
 
 BaseSolver::BaseSolver(MPI_Comm comm, int n, int p) : BaseSolver(comm)
 {
-  m_order = p;
-  m_state.resize(n);
+  order_ = p;
+  state_.resize(n);
 
-  for (auto &state : m_state) {
+  for (auto& state : state_) {
     state = std::make_shared<serac::FiniteElementState>();
   }
 
-  m_gf_initialized.assign(n, false);
+  gf_initialized_.assign(n, false);
```

Comment:
giggle.

Reply:
I am still too lazy to type `grid_function`

Reply:
It should go into the coding guidelines page to NEVER rename this variable. It's one of the few joys i have left.